packages/@okta/i18n/src/properties/login_es.properties

```diff
@@ -856,17 +866,17 @@ oie.okta_verify.totp.enterCodeText = Introduzca el c\u00F3digo de la aplicaci\u0
 oie.okta_verify.push.title = Obtener una notificaci\u00F3n push
 oie.okta_verify.push.sent = Se ha enviado una notificaci\u00F3n push
 oie.okta_verify.push.resend = Reenviar la notificaci\u00F3n push
-oie.okta_verify.signed_nonce.label = Utilizar Okta Verify en este dispositivo
+oie.okta_verify.signed_nonce.label = Use Okta\u00A0FastPass.
```

Comment:
Shouldn't this be "Utilizar Okta\u00A0FastPass"? Also why is a period added?

Reply:
@helenchen-okta "Use" is correct. It's the conjugation of the verb "usar" (to use) in Spanish. See: https://www.spanishdict.com/conjugate/usar

Period is the Spanish language convention. It's uncommon to see sentences without punctuation at the end.